webserver/script/LoadForecasting/LoadForecastingRunner.py

```diff
@@ -45,15 +45,12 @@ def lf_runner(
     total[:, 1] = model.load_segments['Residential L2']['Load']
     total[:, 2] = model.load_segments['Residential MUD']['Load']
     total[:, 3] = model.load_segments['Work']['Load']
-    total[:, 4] = model.load_segments['Fast']['Load'][np.arange(0, 1440, 15)]
+    total[:, 4] = model.load_segments['Fast']['Load'][np.arange(0, 1440, 1)]
     total[:, 5] = model.load_segments['Public L2']['Load']
     total[:, 6] = np.sum(total, axis=1)
     total_df = pd.DataFrame(data=total, columns=['Residential L1', 'Residential L2', 'Residential MUD', 'Work', 'Fast', 'Public L2', 'Total'])
 
     # pd.DataFrame(model.sampled_loads_dict).to_csv('s3://script.forecast.inputsoutputs/Control_Paper_Scenarios/'+name+'_uncontrolled_load.csv')
-
-    model.apply_control(control_rule=work_control, segment='Work')
```

Comment:
Preference to just comment out this single line as we may use it later when the uncontrolled/controlled pairs are implemented for the load forecast algorithm table.

Reply:
Alright I'll add this and merge